endpoints/address/tests/integration_test.js

```diff
@@ -9,3 +10,27 @@ describe('zip', () => {
     request.get(params, resultHandler)
   })
 })
+
+describe('address graphql', () => {
+  it('should return only the field that were asked for', (done) => {
+    const expected = {
+      data: {
+        address: [
+          { street: 'laugavegur', house: '1' },
+        ],
+      },
+    }
+    const params = helpers.testRequestParams('/graphql', { query: `
+      query {
+        address(address: "laugavegur 1") {
+          street
+          house
+        }
+      }
+    ` })
+    request.get(params, (err, res, body) => {
+      assert(body === JSON.stringify(expected), 'result will match expected')
```

Comment:
Oh, I just notice that the second argument to `assert` seems to be "on failure" text.
Which assertion library is this? I don't see anything is `package.json`